pom.xml

```diff
@@ -196,6 +196,12 @@ under the License.
  */
 		</spotless.license.header>
 
+		<!-- This property should contain the add-opens/add-exports commands required for the tests
+		     in the given module to pass.
+		     It MUST be a space-separated list not containing any newlines,
+		     of entries in the form '[-]{2}add-[opens|exports]=<module>/<package>=ALL-UNNAMED'.-->
```

Comment:
> [-]{2}add

This looks as weird as it does because you can't have double-dashes in xml comments :facepalm:

Reply:
It's beautiful